kidkode/src/main/webapp/test_quiz_style.css

```diff
@@ -106,13 +115,35 @@ label {
   padding: 16px 32px;
   text-decoration: none;
```

Comment:
I'm noticing something weird on my end- even when there's no text in the way of the button, the on-hover and click behavior are still weird. The on-hover effect and clicking ability only occur when I move the mouse over the shadow on the bottom of the button, so I'm not able to click on the entire button.

Reply:
Hopefully the position and z-index additions to the button class will have the buttons behaving normally now. Let me know how it is on your end!